src/core/flow.py

```
@@ -0,0 +1,74 @@
+        for task in tasks:
+            chunks, seconds = self.processor.preprocess(task.audio)
+            task.chunks = chunks
+            task.chunk_starts = seconds
+
+
+class ASRHandler(BaseTaskHandler):
+    def __init__(self, asr: "ASRonSPEED", sr: int):
+        self.asr = asr
+        self.sr = sr
+
+    def handle(self, *tasks: AudioTask):
```

Ruff (0.12.2)

Missing return type annotation for public function `handle`

Add return type annotation: `None`

(ANN201)